symplectic_manifolds.tex

```diff
@@ -70,24 +103,52 @@ \section{Symplectic vector spaces}
+	A subspace $W\subset V$ of a symplectic vector space $(V, \Omega)$ is said to be \textbf{symplectic} as well if $(W, \Omega\vert_W)$ is a symplectic vector space. If instead $\Omega\vert_W(u,v) = 0$ $\forall u,v\in W$, the subspace is called \textbf{isotropic}.
+	% and its $\Omega$-orthogonal is then called \textbf{coisotropic}. When a subspace is both, it's named \textbf{Lagrangian}.
 \end{definition}
 
+\begin{example}
+	Let $V=\langle e_1,\dots,e_n,f_1,\dots,f_n\rangle$ where $e_1,\dots,e_n,f_1,\dots,f_n$ is the canonical basis. Then
+	\begin{enumerate}
+		\item	$W=\langle e_{i_1},\dots,e_{i_k},f_{i_1},\dots,f_{i_k}\rangle$ where $k\leq n$, $1\leq i_s\leq n$ and $i_s\neq i_{s'}$ if $s\neq s'$, is a symplectic subspace of V.
+		\item	$W=\langle e_{i_1},\dots,e_{i_k}\rangle$ where $k\leq n$, is a isotropic subspace of V
+	\end{enumerate}
+\end{example}
+
 Thus the decomposition~\eqref{eq:symp_decomp} (when $U = 0$, so $\Omega$ is non-degenerate) is a decomposition of $V$ in minimal symplectic subspaces.
 
 \begin{definition}
-	A \textbf{symplectomorphism} is a linear map $\phi : (V, \Omega) \to (V', \Omega')$ between symplectic vector spaces $(V, \Omega)$ and $(V', \Omega')$ such that
+	A \textbf{symplectomorphism} of symplectic vector spaces $(V, \Omega)$ and $(V', \Omega')$ is a linear isomorphism $\phi : V\overset\sim\longmapsto V' $  such that
 	\begin{eqalign}
-		\Omega'(\phi(u), \phi(v)) = \Omega(u,v), \quad \forall u,v \in V,
+		\phi^* \Omega' = \Omega
 	\end{eqalign}
-	or, alternatively, if $\phi^* \Omega' = \Omega$.
+	where $\phi*$ denotes the pull-back (recall $(\phi^* \Omega')(u,v):=\Omega'(\phi(u), \phi(v))$).
+	If such a $\phi$ exists, $(V, \Omega)$ and $(V', \Omega')$ are called \textbf{symplectomorphic}. 
 \end{definition}
 
-Hence, in the case of symplectic vector spaces, Theorem~\ref{th:decomp_th} says that
+This is equivalent of requiring that following diagram commutes:
+\begin{figure}[H]
+\centering
+	\include{figures/diag-symplectomorphic}
+\end{figure}
+\vspace{-1cm}
+
+\begin{remark}
+The requirement that $\phi$ is an isomorphism is not restrictive, since this is a simple consequence of the trivial kernel of $\Omega$.
+\end{remark}
+
+\begin{remark}
+In the case of symplectic vector spaces, Theorem~\ref{th:decomp_th} says that every symplectic vector space of dimension $2n$ is symplectomorphic to $(\R^{2n},\Omega_0)$:
 \begin{eqalign}
 	(V, \Omega) \underset{\cat{Symp}}\iso (\R^{2n},\Omega_0)
 \end{eqalign}
-where $\Omega_0$ is the symplectic matrix when seen from the canonical basis of $\R^{2n}$.
+with $e_1,\dots,e_n,f_1,\dots,f_n$ the canonical basis of $\R^{2n}$ and $\Omega_0(e_i,e_j)=\Omega_0(f_i,f_j)=0$, $\Omega(e_i,f_j)=\delta_{ij}$, $\forall\,i,j=1,\dots,n$.\\
```

Comment:
Remove this.
Also never use `\\` to start a new line like this! Let LaTeX do its thing with spacing, do not interfere unless it is strictly necessary (in that case, mark it with a comment).